src/ui/popup.js

```diff
@@ -548,7 +552,12 @@ export default class Popup extends Evented {
     }
 
     _getAnchor(offset: any) {
-        if (this.options.anchor) { return this.options.anchor; }
+        if (
+            this.options.fixedAnchor ||
+            (typeof (this.options.fixedAnchor) === 'undefined' && this.options.anchor)
+        ) {
```

Comment:
What if you assign `true` to `fixedAnchor` as a default option? You could write this if as: 

```
if (this.options.anchor && this.options.fixedAnchor) { ... } 
```

Have you tested the case here in which `fixedAnchor = true` and `this.options.anchor = undefined`?

Reply:
But if fixedAnchor is defaulted to `true` wouldn't that change the behaviour for current implementations of having no anchor set? 
Currently if you set no anchor it will default to dynamically moving on screen and favouring the bottom state. Having fixed by default would mean it shouldn't dynamically move. We would be making a breaking change.

> Have you tested the case here in which fixedAnchor = true and this.options.anchor = undefined?
I missed off that test case after changing away from the default params implementation I had before. I've fixed that now.